scripts/Install-WindowsSDK.ps1

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+function Write-Log ([string] $message, $messageColor = "Green") {
+    $currentColor = $Host.UI.RawUI.ForegroundColor
+    $Host.UI.RawUI.ForegroundColor = $messageColor
+    if ($message) {
+      Write-Output "... $message"
+    }
+    $Host.UI.RawUI.ForegroundColor = $currentColor
+}
+
+Push-Location
+try {
```

Comment:
Why do you do that? To stay in the same folder afterwards? What can change it?

Reply:
Haha - Copy paste artifact. (I normally change it to temp folder and download there, I'll use this script in pipelines so removed that functionality. Fixing...)